script.js

```diff
@@ -0,0 +1,245 @@
+const form = document.forms[0];
+const currentTasksBlock = document.getElementById('currentTasks');
+const completedTasksBlock = document.getElementById('completedTasks');
+const currentTasksTitle = document.getElementById('todo-title');
+const completedTasksTitle = document.getElementById('completed-title');
+const inputTitle = document.getElementById('inputTitle');
+const inputText = document.getElementById('inputText');
+const modalLabelText = document.getElementById('exampleModalLabel');
+const submitButton = document.getElementById('submitButton');
+const closeFormIcon = document.getElementById('closeIcon');
+const closeFormButton = document.getElementById('close');
+const increaseSort = document.getElementById('sort-growth');
+const decreaseSort = document.getElementById('sort-wane');
+const modal = document.getElementById('exampleModal');
+const toggleTheme = document.getElementById('toggle-theme');
+const body = document.body;
+let modalEditMode = false;
+let editTaskId = '';
+
+
+const LocalStorageKeys = {
+    tasks: 'tasks',
+    sort: 'sort',
+    darkMode: 'darkMode'
+};
+
+const KeysVariables = {
+    dark: 'dark',
+    transition: 'transition',
+    addTask: 'Add task',
+    editTask: 'Edit task',
+    save: 'Save Changes',
+    editBtn: 'btn-info',
+    changeStatusBtn: 'btn-success',
+    yellowFont: 'bg-warning',
+    greenFont: 'bg-success',
+    redFont: 'bg-danger',
+    lowPriority: 'Low',
+    mediumPriority: 'Medium'
+};
```

Comment:
You did right by using object, but It looks not very useful when everything is in one big object. Here we can also say that single responsibility is violated. Split `KeysVariables` by meaning in several groups.

Reply:
`const KeysVariables = {
    theme: {
        dark: 'dark',
    },
    text: {
        addTask: 'Add task',
        editTask: 'Edit task',
        save: 'Save Changes',
    },
    styles: {
        transition: 'transition',
        yellowFont: 'bg-warning',
        greenFont: 'bg-success',
        redFont: 'bg-danger',
        lowPriority: 'Low',
        mediumPriority: 'Medium'
    },
    buttons: {
        editBtn: 'btn-info',
        changeStatusBtn: 'btn-success',
    }
};`
Fixed

Reply:
```suggestion
const KeysVariables = {
    theme: {
        dark: 'dark',
    },
    text: {
        addTask: 'Add task',
        editTask: 'Edit task',
        save: 'Save Changes',
    },
    styles: {
        transition: 'transition',
        yellowFont: 'bg-warning',
        greenFont: 'bg-success',
        redFont: 'bg-danger',
        lowPriority: 'Low',
        mediumPriority: 'Medium'
    },
    buttons: {
        editBtn: 'btn-info',
        changeStatusBtn: 'btn-success',
    }
};
```

Reply:
Please use PascalCase or Snake Case (All Caps) - USER_LOGIN_COUNT for constants. It's common practice to distinguish such constants(enum) from common objects.

Reply:
```suggestion
const THEME_CLASSNAMES = {
    dark: 'dark',
    transition: 'transition'
}
const TEXT_VARIABLES = {
    addTask: 'Add task',
    editTask: 'Edit task',
    save: 'Save Changes',
}
const PRIORITIES = {
    low: 'Low',
    medium: 'Medium',
    high: 'High'
}
const TASK_CLASSNAMES = {
    yellowFont: 'bg-warning',
    greenFont: 'bg-success',
    redFont: 'bg-danger',
}
const BUTTON_CLASSNAMES = {
    editBtn: 'btn-info',
    changeStatusBtn: 'btn-success',
}
```
Is it ok now?